docs/about-us.ftd

```diff
@@ -25,9 +25,6 @@ sections: $lib.sitemap.sections
 
 -- ds.page.body:
 
-We have developed the `about-us` component for [admint.io/about](https://admint.io/#/about), 
-which matches the design currently displayed on the live site.
-
 -- ds.h3: How to use?
```

Comment:
```suggestion
-- ds.h1: How to use?
```

Please confirm this suggestion once with AmitU, he suggests to use `h1`.
Also, add some content between Page title and `-- ds.h1` or `-- ds.h3`